readme.txt

```diff
@@ -2,10 +2,10 @@
 === Sensei LMS - Online Courses, Quizzes, & Learning ===
 Contributors: automattic, alexsanford1, burtrw, donnapep, gikaragia, jakeom, merkushin, m1r0, renathoc, yscik
```

Comment:
Not very important I guess, but I just observed that neither I nor @aaronfc are on this list: 

```suggestion
Contributors: automattic, alexsanford1, burtrw, donnapep, gikaragia, jakeom, merkushin, m1r0, renathoc, yscik, fjorgemota, aaronfc
```